### PR TITLE
Add support for textures on MergedMesh and update to I3DFv7

### PR DESCRIPTION
**Pull request by dragly**

### CODE REVIEW

File: src/geometry/MergedMeshGroup.ts [hunk 1]
```diff
@@ -158,11 +159,29 @@ export class MergedMesh {
...
-  constructor(capacity: number, fileId: number, createdByInstancedMesh: boolean = false) {
+  geometry?: THREE.BufferGeometry;
+  diffuseTexture?: TextureInfo;
+  specularTexture?: TextureInfo;
+  ambientTexture?: TextureInfo;
+  normalTexture?: TextureInfo;
+  bumpTexture?: TextureInfo;
+  constructor(capacity: number, fileId: number,
+    createdByInstancedMesh: boolean = false,
+    diffuseTexture?: TextureInfo,
```
Comment (ghost) [inline src/geometry/MergedMeshGroup.ts:172]: will the amount of textures increase? should we make this an object so its not as large as a constructor :P

File: src/parsers/i3d/loadUncompressedValues.ts [hunk 2]
```diff
@@ -48,6 +48,18 @@ export default function loadUncompressedValues(fileReader: CustomFileReader) {
...
+      case 'texture':
+        if (bytesForOneValue !== 16) {
+          throw Error('Reading incorrect number of bytes for fileId');
+        }
+        for (let j = 0; j < clusterCount; j++) {
+          const fileId = fileReader.readUint64();
+          const width = fileReader.readUint16();
+          const height = fileReader.readUint16();
+          const _reserved = fileReader.readUint32();
+          uncompressedValues.texture!.push({fileId: fileId, width: width, height: height});
```
Comment (ghost) [inline src/parsers/i3d/loadUncompressedValues.ts]: nitpick: this missed liniting

**Review by ghost — APPROVED** [review]
Some comments, looks clean overall :D

File: src/parsers/i3d/loadUncompressedValues.ts (hunk 2 above)
Comment (ghost) [reply src/parsers/i3d/loadUncompressedValues.ts]: wow youre fast

File: src/geometry/MergedMeshGroup.ts (hunk 1 above)
Comment (dragly) [reply src/geometry/MergedMeshGroup.ts:172]: No, should be these for the time being. But I totally agree. If there are more incoming at some point, we can create a separate object.

**Pull request merged**